matflow_defdap/snippets/link_EBSD_DIC_maps.py

```diff
@@ -2,11 +2,11 @@
 
 
 @main_func
-def link_EBSD_DIC_maps(EbsdMap, DicMap, dic_homog_points, dic_min_grain_size,
-                       ebsd_homog_points, transform_type):
+def link_EBSD_DIC_maps(ebsd_map, dic_map, dic_homog_points, dic_min_grain_size,
+                       ebsd_homog_points, transform_type, find_grains_algorithm):
 
-    DicMap.homogPoints = dic_homog_points
-    EbsdMap.homogPoints = ebsd_homog_points
-    DicMap.linkEbsdMap(EbsdMap, transformType=transform_type)
+    dic_map.homogPoints = dic_homog_points
+    ebsd_map.homogPoints = ebsd_homog_points
+    dic_map.linkEbsdMap(ebsd_map, transformType=transform_type)
 
-    DicMap.findGrains(minGrainSize=dic_min_grain_size)
+    dic_map.findGrains(minGrainSize=dic_min_grain_size,find_grains_algorithm=find_grains_algorithm)
```

Comment:
Should this be: 
```
dic_map.findGrains(minGrainSize=dic_min_grain_size, algorithm=find_grains_algorithm)
```
? https://github.com/MechMicroMan/DefDAP/blob/4b49443a8f0c3ac97d30dd30a333e3797cdda27f/defdap/hrdic.py#L824